com.ibm.streamsx.metrics/impl/java/src/com/ibm/streamsx/metrics/internal/TupleContainer.java

```diff
@@ -40,7 +40,7 @@
 	private Integer _jobNameAttributeIndex = null;
```

Comment:
It seems to me as if this file can be taken from master as-is. See my other comments for this file.